frontend/src/plaquettes/Template.js

```diff
@@ -247,14 +104,22 @@ export default class Template {
       this.selectedQubits = this.selectedQubits.filter((q) => q !== qubit);
       return;
     }
+    qubit.changeColor('red');
     this.selectedQubits.push(qubit);
+    // Change the color of the qubit
+    // Check if the selected qubits are 3 or more
     if (this.selectedQubits.length > 2) {
       // Show the button
-      this.plaquetteButton.visible = true;
+      this.selectQubitsButton.visible = false;
+      this.createPlaquetteButton.visible = true;
     }
   };
 
-  // Create the plaquette from the selected qubits and assign it to the template
+  /**
+   * Create the plaquette from the selected qubits and assign it to the template
```

Comment:
assign it to the `Footprint`? Again, I still need to understand the difference.

Reply:
Not sure what you mean here....I have made the name change though :)

Reply:
The notion of a plaquette is different from a plaquette footprint, and the goal of the name change is to make that clear. We should discuss in the meeting, I think.

Reply:
That's clear to me. I wasn't sure if you were commenting on the edit from single line comment to multiline

Reply:
I'm saying it should say `assign it to the footprint`